WebRoot/js/zimbraMail/mail/controller/ZmComposeController.js

```diff
@@ -1228,7 +1228,7 @@ function() {
 		buttons.push(ZmOperation.SEND);
 	}
 
-	buttons.push(ZmOperation.CANCEL, ZmOperation.SEP, ZmOperation.SAVE_DRAFT);
+	buttons.push(ZmOperation.SAVE_DRAFT, ZmOperation.SEP, ZmOperation.CANCEL);
```

Comment:
what does ZmOperation.SEP do here? can we remove if not required
not sure are we maintaining this in other places

@tarangkhandelwal @kalpeshhpatel let me know your thoughts about this